src/components/grid/grid.ts

```diff
@@ -471,8 +471,8 @@ export class Grid extends Component<Props, SpreadsheetChildEnv> {
     const lastZone = zones[zones.length - 1];
     let type: ContextMenuType = "CELL";
     if (!isInside(col, row, lastZone)) {
-      this.env.model.dispatch("UNFOCUS_SELECTION_INPUT");
-      this.env.model.dispatch("STOP_EDITION");
+      this.env.model.dispatch("STOP_EDITION"); // to keep the content
```

Comment:
this should be handled by the `getBacktoDefault` function. the release callback set in the edition plugin ( https://github.com/odoo/o-spreadsheet/blob/master/src/plugins/ui_stateful/edition.ts#L418 ) should do the same thing that the `dispatch("stop_edition")` does. you can probably extract the logic in a single function.